netperf_tool/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>netperf_tool</name>
+  <version>0.19.0</version>
+  <description>
+    Simple iperf like tool for ros2.
+  </description>
+
+  <maintainer email="aditya.pande@openrobotics.org">Aditya Pande</maintainer>
+  <maintainer email="audrow@openrobotics.org">Audrow Nash</maintainer>
+  <maintainer email="michael.jeronimo@openrobotics.org">Michael Jeronimo</maintainer>
```

Comment:
Is this intentional due to it being in `ros2cli`?

Reply:
Yes, I always copy the maintainers of the repo.
We could create a new repo for this if that's preferred though.
None of the packages I'm adding need to live in this repo.

Reply:
Personally, I think it would be better to have this in an external repository.  There are a few reasons for that:

1. As it stands, the `ros2cli` repository is purely Python.  While we don't *have* to maintain that, it does make it easier to for maintainers to only have to deal with a single language in a repository.
2. If we put it in an external repository, we can still release it into the distributions, but it doesn't necessarily need to be part of the core.  That allows it to evolve independently of the core, and also makes it so that it isn't release-critical.

That said, if others feel strongly that it should be here, I can be persuaded otherwise.

Reply:
I don't mind stronly either way.
I opened a PR targetting this repo only as a first step.

If we agree to move it to another repo, I think it would still be good to finish addressing all the comments here and then move the code to another repo.

Reply:
I'm in favor of putting it in a separate repository for the same reasons as @clalancette, but it does add some support overhead. Thought I suppose that is reduced slightly now that we have rolling.